src/lib/components/dashboard/teams/TeamsList.svelte

```diff
@@ -53,52 +54,54 @@
       </Tooltip>
     </div>
   </div>
-  <div class="sidebar-teams-list sparrow-thin-scrollbar overflow-y-auto">
-    {#each teams as team, index}
-      <button
-        class={`d-flex w-100 align-items-center justify-content-between rounded teams-outer border-0 ${
-          currOpenedTeam.id == team.teamId && "active"
-        }`}
-        on:click={async () => {
-          handleOpenTeam(team.teamId, team.name, team.logo);
-          await teamRepositoryMethods.setOpenTeam(team.teamId);
-          if (team.isNewInvite) {
-            let data = await teamServiceMethods.disableNewInviteTag(
-              userId,
-              team.teamId,
-            );
-            if (data) {
-              data.isNewInvite = false;
-              teamRepositoryMethods.modifyTeam(team.teamId, data);
+  <div class="sidebar-teams-list">
+    <List height={"calc((100vh - 230px) / 3)"} classProps={"px-3 py-0"}>
+      {#each teams as team, index}
+        <button
+          class={`d-flex w-100 align-items-center justify-content-between rounded teams-outer border-0 ${
+            currOpenedTeam.id == team.teamId && "active"
+          }`}
+          on:click={async () => {
+            handleOpenTeam(team.teamId, team.name, team.logo);
+            await teamRepositoryMethods.setOpenTeam(team.teamId);
+            if (team.isNewInvite) {
+              let data = await teamServiceMethods.disableNewInviteTag(
+                userId,
+                team.teamId,
+              );
+              if (data) {
+                data.isNewInvite = false;
+                teamRepositoryMethods.modifyTeam(team.teamId, data);
+              }
             }
-          }
-        }}
-      >
-        <div class="d-flex w-100 overflow-hidden">
-          {#if base64ToURL(team.logo) == "" || base64ToURL(team.logo) == undefined}
-            <p
-              class={`m-0 text-defaultColor me-2 align-items-center justify-content-center bg-transparent border-defaultColor `}
-              style={`font-size: 15px; padding-top: 2px; width: 25px !important; height: 25px !important; display: flex; border: 1px solid #45494D; border-radius: 50%;`}
-            >
-              {team.name[0] ? team.name[0].toUpperCase() : ""}
+          }}
+        >
+          <div class="d-flex w-100 overflow-hidden">
+            {#if base64ToURL(team.logo) == "" || base64ToURL(team.logo) == undefined}
+              <p
+                class={`m-0 sparrow-fs-15 text-defaultColor me-2 align-items-center justify-content-center bg-transparent border-defaultColor `}
+                style={`padding-top: 2px; width: 25px !important; height: 25px !important; display: flex; border: 1px solid #45494D; border-radius: 50%;`}
```

Comment:
absolute width and height

Reply:
It is fine for now as said by @parthdalaltech